### DISCUSSION TITLE
Adding Hyperlink on the Recommendation Tile

### DISCUSSION THREAD

**Discussion by SanchanaMohankumar** [microsoft/Microsoft-Fabric-workload-development-sample#27] [Q&A] [answered]
As we can see in Image, we are unable to add a hyperlink. We also tried to add it, as shown in Image, but still received no response. Please do provide your input.

Image 
The image has concealed path details for confidentiality
<img width="686" alt="Index worker ts edited" src="https://github.com/microsoft/Microsoft-Fabric-workload-development-sample/assets/175164325/2a289a91-825d-4cda-868e-c291182074e6">

**Comment by anitamayorov** [comment]
Hi @SanchanaMohankumar,

The learningMaterials section in FE manifests is unrelated to the recommendations section - can you please clarify what is the issue you are experiencing? 
Regarding learningMaterials - we are investigating the issue, no ETA for fix.

**Reply by jainr** [reply]
@anitamayorov - What is the learning material section of FE manifest used for?

**Reply by anitamayorov** [reply]
@jainr, 
The learning material section can be used to open notifications, other actions registered in index.worker, and even open a link to suggest further reading. Regarding the links, it is currently an issue we are investigating, no ETA yet.

**Comment by axel92b** [comment, marked as answer]
Hi  @SanchanaMohankumar,
Hyperlink support for Recommendation Tiles should arrive the week of September 21.

**Reply by MikeCarlo** [reply]
is this fixed yet?  

Still not seeing this working as of 10/11/2024

**Reply by q-benwillis** [reply]
Still not working as of 31/10/2024

I've tried using js to open a webpage but this is blocked:

> main.worker.ts:24 Blocked opening 'https://contoso.com/help' in a new window because the request was made in a sandboxed frame whose 'allow-popups' permission is not set.

**Reply by anitamayorov** [reply]
Hi @q-benwillis and @MikeCarlo , 

The fix is already supported, and we will publish the updated sample code and documentation next week. In the meanwhile, you can already support it by applying the following changes in your Product.json file:
1. Remove 
```
          "onClick": {
            "action": "open.learningMaterial.dialog2"
          }
```
2. Add instead
```
          "link": "https://aka.ms/Fabricdeveloperdocumentation"
```

Regarding the popup, the iframe is sandboxed and allow-popups is not supported, you can find the documentation on this here:  https://learn.microsoft.com/en-us/fabric/workload-development-kit/extensibility-front-end#:~:text=Note,MDN%20web%20docs
